### DISCUSSION TITLE
Preference custom photos over Trefle

### DISCUSSION THREAD

**Discussion by 0099FF** [MDeLuise/plant-it#54] [Ideas]
I'd like to see my own custom photos in the home screen rather than the ones supplied by Trefle (just like I can on individual plant screens).

**Comment by MDeLuise** [comment]
Hi @0099FF, thanks for the feedback.

It's a good idea, I'm thinking of creating a "thumbnail image" feature for each user's plant.
It could be chosen from the uploaded images for that plant, or it could be provided via URL.
That image will be displayed in the homescreen of the app for that plant.

It could be also useful to have a feature like "use always the last uploaded plant image as thumbnail", if so in the homescreen it will be shown not the Trefle image but the last uploaded from the user.

**Reply by 0099FF** [reply]
That sounds fantastic. Another interesting option would be a "random" user uploaded image each time the homescreen is loaded 😃

**Reply by MDeLuise** [reply]
Another great idea, thanks 😁
I created #55 in order to track this feature, as soon as possible I will start working on that

**Reply by MDeLuise** [reply]
Hi @0099FF,
With the [0.0.17](https://github.com/MDeLuise/plant-it/releases/tag/0.0.17) release of the project this feature was added 😁.
Please note that there is also a newest version of the system, `0.1.0`.
